src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -171,7 +171,7 @@ You are encouraged to order app shortcuts by priority, with the most critical
 app shortcuts appearing first in the `shortcuts` array as the limit on the
 number of app shortcuts displayed varies depending on the platform. Chrome and
 Edge on Windows for instance limit the number of app shortcuts to 10 while
-Chrome for Android only takes the first 4 app shortcuts into account.
+Chrome for Android only takes the first 3 app shortcuts into account.
```

Comment:
Can you please add that this changed in Chrome 92 and say something about why? This would help people who are trying to understand why their 4th shortcut is suddenly failing.

Reply:
Done.